doc/Overview/Mvux/ListFeeds.md

```diff
@@ -0,0 +1,77 @@
+---
+uid: Overview.Mvux.ListFeeds
+---
+
+# What are List-Feeds?
+
+The `IListFeed<T>` is like a [Feed](xref:Overview.Mvux.Feeds) which is stateless and keeps no track of changes, but is specialized for handling collections.
+
+In Feeds, each data request from the service, returns one single item, whereas in List-Feed each request returns a collection of items.  
+Its distinctive feature is the ability to use its [Operators](xref:Overview.Mvux.ListFeeds#operators) directly on its items, rather than on the entire collection.
+
+Another important thing to remember is that when the service returns an empty collection, it's treated as an Empty message, and the data axis Option will be `None`, although the result was not `null`.
+Read this if you would like to review the [Feed Messages section](xref:Overview.Mvux.Feeds#messages).
+
+
+> [!NOTE]  
+> The List-Feed is using the _key equality_ to track multiple version of a same entity within different messages of the List-Feed.
+(Read more about _key equality_.)[xref:Overview.KeyEquality.Concept]
+
+## How to create a list feed
+
+To create an `IListFeed<T>`, use the the static class `ListFeed` to call one of the same `Async`, `AsyncEnumerable` and `Create` methods as in Feed.  
+The only difference is that it expects the Task or Async Enumerable to return an `IImmutableList<T>` instead of `T`.
```

Comment:
@dr1rrb is there a reason why we've opted for requiring IImmutableList<T> rather than the much wider IEnumerable<T>?